app/routes/crate/version-dependencies.js

```diff
@@ -1,25 +1,39 @@
+import { NotFoundError } from '@ember-data/adapter/error';
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 
 export default class VersionRoute extends Route {
+  @service store;
   @service router;
 
   async model(params, transition) {
     let crate = this.modelFor('crate');
 
-    let versions;
+    // TODO: Resolved version without waiting for versions to be resolved
     try {
-      versions = await crate.loadVersionsTask.perform();
+      await crate.loadVersionsTask.perform();
     } catch (error) {
       let title = `${crate.name}: Failed to load version data`;
       return this.router.replaceWith('catch-all', { transition, error, title, tryAgain: true });
     }
```

Comment:
do we still need this? 🤔

Reply:
Ah, yeah, this could be removed! It should be able to auto-load if needed (primarily for version counts).

Reply:
Ah, I remember why I kept this, as we have an acceptance case for versions. See: https://github.com/rust-lang/crates.io/actions/runs/13137385348/job/36655968746?pr=10344#step:6:90. I think we can remove this test case now.